### PR TITLE
Log bundle registration scope.

### PR DESCRIPTION
**Pull request by barnson**

### CODE REVIEW

File: src/internal/SetBuildNumber/Directory.Packages.props.pp [hunk 1]
```diff
@@ -53,7 +53,7 @@
     <PackageVersion Include="System.Reflection.Metadata" Version="1.8.1" />
     <PackageVersion Include="System.Security.Principal.Windows" Version="4.7.0" />
     <PackageVersion Include="System.Text.Encoding.CodePages" Version="4.7.1" />
-    <PackageVersion Include="System.Text.Json" Version="6.0.9" />
+    <PackageVersion Include="System.Text.Json" Version="6.0.10" />
```
Comment (robmen) [inline src/internal/SetBuildNumber/Directory.Packages.props.pp:56]: Would be good to put this in a separate commit.

**Review by robmen — CHANGES_REQUESTED** [review]

File: src/internal/SetBuildNumber/Directory.Packages.props.pp (hunk 1 above)
Comment (barnson) [reply src/internal/SetBuildNumber/Directory.Packages.props.pp:56]: Done.

**Comment by barnson** [comment]
Stand by.

**Review by robmen — APPROVED** [review]
